frontend/src/component/common/StrategyItemContainer/StrategyItemContainer.tsx

```diff
@@ -74,6 +74,20 @@ const StyledContainer = styled(Box, {
     background: disabled
         ? theme.palette.envAccordion.disabled
         : theme.palette.background.paper,
+    '& p, & span, & h1, & h2, & h3, & h4, & h5, & h6': {
```

Comment:
Make all text elements in StrategyExecution light grey if disabled

Reply:
is it to avoid styling child elements individually?

Reply:
Yes